src/pages/disease-catalog/index.md

```diff
@@ -54,6 +54,6 @@ acknowledgements_block:
 funding_text: >
   We wish to thank Allen Institute founders, Jody Allen & Paul G. Allen, for their vision, encouragement, and support.
 ---
-The Disease Collection Cell Catalog is a growing compilation of cell lines that carry mutations in genes known to cause disease. These cell lines were created by introducing a point mutation in one of the fluorescently tagged WTC-11 clonal lines from the [Allen Cell Collection](https://www.allencell.org/cell-catalog.html).
+The Disease Collection Cell Catalog is a growing compilation of cell lines that carry mutations in genes known to cause disease. These cell lines were created by introducing a point mutation in one of the fluorescently tagged WTC-11 clonal lines from the [Allen Cell Collection](https://cell-catalog.allencell.org/).
```

Comment:
ideally, both this, and the link to the disease catalog would be to the current domain. Ie, if we're running in dev it should go to localhost. Can you try just having the link be "/" for this and see if it works? I know it's a little strange since this is md

Reply:
Good call! thanks for pointing that out, I updated both links to use relative paths and they worked well. I'll keep this in mind going forward